### PR TITLE
Added checkbox widget with state changed on click

### PR DESCRIPTION
**Pull request by marcin-witaszek**

### CODE REVIEW

File: cogboard-webapp/src/components/styled/index.js [hunk 1]
```diff
@@ -14,6 +14,17 @@ export const WidgetButton = styled(FullWidthButtonOutlined)`
   margin-top: auto;
 `;
 
+export const WidgetIconButton = styled(IconButton)`
+  position: absolute;
```
Comment (goeson00) [inline cogboard-webapp/src/components/styled/index.js]: It is just a matter of preference but here we sort css props alphabetically.

File: cogboard-webapp/src/components/widgets/types/CheckboxWidget.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  };
+
+  return (
+    <WidgetIconButton aria-label="Toggle status" onClick={handleChangeStatus}>
+      <StatusIcon size="large" status={status}/>
+    </WidgetIconButton>
+  );
+};
+
+const StatusIcon = ({ status, size }) => {
```
Comment (goeson00) [inline cogboard-webapp/src/components/widgets/types/CheckboxWidget.js]: Move it to separate file.

File: cogboard-webapp/src/components/widgets/types/CheckboxWidget.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    FAIL: Close,
+    UNKNOWN: IndeterminateCheckBoxOutlined
+  };
+
+  const IconComponent = status in iconComponents ? iconComponents[status] : iconComponents['UNKNOWN'];
+
+  return <IconComponent fontSize={size} />;
+};
+
+const getNextStatus = (status) => {
```
Comment (goeson00) [inline cogboard-webapp/src/components/widgets/types/CheckboxWidget.js]: Move it to helpers.js inside CheckboxWidget folder. Use WidgetTypeForm for structural reference.

File: cogboard-webapp/src/components/widgets/types/CheckboxWidget.js [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+  const statusArray = ['OK', 'FAIL', 'UNKNOWN'];
+  const statusIndex = statusArray.indexOf(status);
```
Comment (goeson00) [inline cogboard-webapp/src/components/widgets/types/CheckboxWidget.js]: I would rather combine it with newer array methods like find and findIndex than indexOf, but it is ok.

File: cogboard-webapp/src/components/styled/index.js (hunk 1 above)
Comment (marcin-witaszek) [reply cogboard-webapp/src/components/styled/index.js]: Ok, done

File: cogboard-webapp/src/components/widgets/types/CheckboxWidget.js (hunk 2 above)
Comment (marcin-witaszek) [reply cogboard-webapp/src/components/widgets/types/CheckboxWidget.js]: Sure, done

File: cogboard-webapp/src/components/widgets/types/CheckboxWidget.js (hunk 3 above)
Comment (marcin-witaszek) [reply cogboard-webapp/src/components/widgets/types/CheckboxWidget.js]: Done

File: cogboard-webapp/src/components/widgets/types/CheckboxWidget.js (hunk 4 above)
Comment (marcin-witaszek) [reply cogboard-webapp/src/components/widgets/types/CheckboxWidget.js]: Ok, done

**Pull request merged**